redirects-transition.conf

```diff
@@ -1,5 +1,12 @@
 #Redirect rules for specific pages
 
+
+rewrite ^/pages/brochures/pubfund_limits_2016.shtml https://www.fec.gov/help-candidates-and-committees/understanding-public-funding-presidential-elections/presidential-spending-limits-2016/ redirect;
+rewrite ^/pdf/record/1999/index99_1.htm https://www.fec.gov/resources/cms-content/documents/index1999.pdf redirect;
+rewrite ^/pdf/record/1998/index98_1.htm https://www.fec.gov/resources/cms-content/documents/index1998.pdf redirect;
+rewrite ^/pdf/record/1997/index97_1.htm https://www.fec.gov/resources/cms-content/documents/index1997.pdf redirect;
+rewrite ^/pages/brochures/jointfundraising_brochure.pdf https://www.fec.gov/help-candidates-and-committees/making-disbursements/joint-fundraising-candidates-political-committees/ redirect;
+rewrite ^/rad/documents/FECFileGettingStartedManual.pdf https://www.fec.gov/resources/cms-content/documents/FECFile_GettingStartedManual_Candidates.pdf
```

Comment:
@kathycarothers You're missing this ` redirect;` at the end of this redirect on line 9

Reply:
Fixed now @patphongs

Reply:
Added in a redirect suggested by https://github.com/fecgov/FEC/issues/9579